contents/blog/titles.md

```diff
@@ -0,0 +1,79 @@
+
+## The most difficult title to spec out
+
+It was tough hiring our first “Designer.”
+
+We’re big believers in self-serve products and experiences, and our website and docs are our storefront - so we wanted someone that would throw themselves into our website.
+
+But because we're not a design company, we couldn’t figure out if we needed a UX/UI designer, a frontend developer, a website designer, or a general designer. Many folks in the interview process told us we were asking for too much - but sometimes that's just what your early-stage startup needs.
+
+It took lots of iteration in our hiring process to get this right. We ended up hiring [Cory](https://www.linkedin.com/in/corywatilo/), a talented generalist who can take designs from concept to wireframe to execution. He now works across our website, docs, and the app.
+
+> _"Great designers iterate. They don’t magically come up with something brilliant on the first try."_ - [Malthe Sigurdsson](https://www.linkedin.com/in/malthe/), former Chief Design Officer at Stripe.
+
+Ex-founders or early employees make for brilliant generalists. They’re less likely to be as deeply skilled in any given area, but if you need a quick uplift in a certain area, they’ll get you there faster and more flexibly. As your company grows and your requirements change, you can specialize your roles and titles. For example, as of writing, we're hiring for a Technical Blogger and Developer Educator to bolster our content and developer relations efforts. (Check out our [Career](https://posthog.com/careers) page for more open roles.)
+
+## Even the CEO's job changes over time
+
+[James](https://www.linkedin.com/in/j-hawkins/), our CEO, has had to wear many hats over the years. Although his role swings between dealing with customers, recruiting, fundraising, and marketing, he's always been focused primarily on selling with a bit of design on the side.
+
+Before PostHog started, James had to sell to get Tim to join. After the company launched, he had to get early users on board, which meant a mix of selling and building our website. His role then cycled through getting investors to take part in our funding rounds (from seed to [Series B](https://twitter.com/posthog/status/1402988140332154888)), hiring people, and talking to customers.
+
+On the design side, it involves thinking about how the company and our business model should work - including our [product](https://posthog.com/handbook/strategy/roadmap), financing, [deployment](https://posthog.com/docs/self-host), [team structure](https://posthog.com/handbook/people/team-structure/team-structure), and [pricing](https://posthog.com/pricing) strategies.
```

Comment:
I wouldn't say this is design thinking, especially as it could be confused with the example above from Cory who _is_ a designer. It's more Ops (financing, org, etc) and Product (roadmap, pricing, deployment).